test/Groups/Permutations.jl

```diff
@@ -427,3 +427,24 @@ end
   H,iso = smaller_degree_permutation_representation(G)
   @test degree(H)<degree(G)
 end
+
+@testset "fixed_points tests" begin
+  # Setup permutation group
+  g = symmetric_group(4)             # S₄
+  s = sylow_subgroup(g, 3)[1]        # a Sylow 3-subgroup (order 3)
+  e = one(g)                         # identity element
+  y = gens(g)[1]                     # an example element of g
+  x = gens(s)[1]                     # generator of Sylow subgroup
+
+  # Tests for fixed_points on individual elements
+  @test fixed_points(x) == [4]       # Sylow generator fixes only 4
+
+  @test fixed_points(e) == collect(1:degree(g))  # Identity fixes all
+
+  @test sort(fixed_points(y)) ==
+      sort([i for i in 1:degree(g) if y(i) == i])  # Check example element
```

Comment:
I don't think you need `sort` here?
```suggestion
  @test fixed_points(y) == [i for i in 1:degree(g) if y(i) == i]
```
(You could also use `issetequal` but I don't see the need?)

Reply:
I was just being cautious, I've implemented the change you suggested. I've also added number of fixed points, please let me know for any corrections.

Reply:
adding to this, I implemented this suggestion after I pushed, apologies for that. I will just push it again together with any other changes required for number of fixed points.